src/main/kotlin/com/cognifide/gradle/aem/deploy/tasks/PurgeTask.kt

```diff
@@ -0,0 +1,28 @@
+package com.cognifide.gradle.aem.deploy.tasks
+
+import com.cognifide.gradle.aem.AemPlugin
+
+open class PurgeTask : AbstractTask() {
+
+    companion object {
+        val NAME = "aemPurge"
+    }
+
+    init {
+        group = AemPlugin.TASK_GROUP
+        description = "Uninstalls and then deletes AEM package on instance(s)."
+    }
+
+    @org.gradle.api.tasks.TaskAction
```

Comment:
IntelliJ went weird while moving classes around and removed all imports in favor of fully qualified names. I overlooked this one.